### ISSUE TITLE
Issue in getting response from Dialogflow

### ISSUE THREAD

**Issue by cloudnabeel** [actions-on-google/actions-on-google-nodejs#384]
This is the URL( https://dialogflow.googleapis.com/v2/{parent=projects/<projectId>}/agent) in my case( https://dialogflow.googleapis.com/v2/{parent=projects/chatagent-yrwptu}/agent) which I want to access from the POSTMAN tool to check its response whether is working or not. But every time it is giving me an error 
<p>The requested URL <code>/v2/%7Bparent=projects/chatagent-yrwptu%7D/agent</code> was not found on this server.
<ins>That’s all we know.</ins>
I seriously don't know what to do

**Comment by Fleker** [comment]
When you see `{parents=projects/chatagent-yrwptu}` it does not mean literally `parents=`. It's meant to signify that the URL path for the v2 API is `agent`, but it needs to be prepended by a `parent` so that the API knows which project specifically to use.

So you should be using the URL https://dialogflow.googleapis.com/v2/projects/chatagent-yrwptu/agent

**Comment by cloudnabeel** [comment]
Yes I was using this URL from Salesforce org but it was not giving me any response, so what should I do to get the response in my salesforcr org

**Comment by cloudnabeel** [comment]
I have tried this code but I was confused in this curl query
`Http http = new Http();
HttpRequest request = new HttpRequest();
request.setHeader()
request.setEndpoint('curl\-H "Authorization: Bearer 2041141a7ad34f45ae12654480ac0c98"\"https://dialogflow.googleapis.com/v2/{parent=projects/chatagent-yrwptu}/agent/query?v=20150910&contexts=shop&lang=en&query=apple&sessionId=12345&timezone=America/New_York"');
request.setMethod('Get');
HttpResponse response = http.send(request);
String body = response.get(body);
System.debug(body);
`

**Comment by Fleker** [comment]
Your cURL command has the same issue where you're passing `parent=`. If you use the standard URL you will get a different answer. However, I also am not seeing any documentation for the approach you're taking in the request.

**Comment by cloudnabeel** [comment]
Thanku @Fleker for your quick response on this issue
